010-AzureDevOps/Coach/Solution-01.md

```diff
@@ -4,17 +4,5 @@
 
 ## Notes & Guidance
 
-This is the only section you need to include.
-
-Use general non-bulleted text for the beginning of a solution area for this challenge
-
-- Then move into bullets
-  - And sub-bullets and even
-    - sub-sub-bullets
-
-Break things apart with more than one bullet list
-
-- Like this
-- One
-- Right
-- Here
+- Provide access to the team across the organization.
+- DevOps Self- Assessment with issue in the certificate.
```

Comment:
Since there is an issue with the certificate, is this assessment being pulled or will a new link be provided?

Reply:
I added itens in the file fo alerts users or just skip the step.